image_proc/launch/image_proc.launch.py

```diff
@@ -30,36 +30,82 @@
 # ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
 # POSSIBILITY OF SUCH DAMAGE.
 
-from launch import LaunchDescription
-from launch_ros.actions import ComposableNodeContainer
+import sys
+
+from launch import LaunchDescription, LaunchIntrospector, LaunchService
+from launch_ros import actions, get_default_launch_description
+
 from launch_ros.descriptions import ComposableNode
 
 
 def generate_launch_description():
-    return LaunchDescription([
-        ComposableNodeContainer(
-            package='rclcpp_components', node_executable='component_container',
-            node_name='image_proc_container', node_namespace='',
-            composable_node_descriptions=[
-                ComposableNode(
-                    package='image_proc',
-                    node_plugin='image_proc::DebayerNode',
-                    node_name='debayer'
-                ),
-                ComposableNode(
-                    package='image_proc',
-                    node_plugin='image_proc::RectifyNode',
-                    node_name='rectify_mono'
-                ),
-                ComposableNode(
-                    package='image_proc',
-                    node_plugin='image_proc::RectifyNode',
-                    node_name='rectify_color',
-                    remappings=[
-                        ('image_mono', 'image_color'),
-                        ('image_rect', 'image_rect_color')
-                    ]
-                ),
-            ],
-        ),
-    ])
+    ld = LaunchDescription()
+
+    # Load composable container
+    image_processing = actions.ComposableNodeContainer(
+        node_name="image_proc_container",
+        package='rclcpp_components',
+        node_executable='component_container',
+        composable_node_descriptions=[
+            ComposableNode(
+                package='image_proc',
+                node_plugin='image_proc::DebayerNode',
+                node_name='debayer_node',
+            ),
+            # Example of rectifying an image
+            ComposableNode(
+                package='image_proc',
+                node_plugin='image_proc::RectifyNode',
+                node_name='rectify_mono_node',
+                # Remap subscribers and publishers
+                remappings=[
+                    # Subscriber remap
+                    ('image', 'image_mono'),
+                    ('camera_info', 'camera_info'),
+                    # Publisher remap
+                    ('image_rect', 'image_rect')
```

Comment:
These two remaps aren't actually remapping anything so they can be removed.

Reply:
@JWhitleyWork since this is a demo, I think its useful to show all 3 of them even if right now they don't do anything. Lets someone easily update this for their applications.

Reply:
Either way is fine with me. I'm not picky on this one - I just thought they were unnecessary. @caelinsutch, would you mind adding them back? Sorry about the back-and-forth.